### ISSUE TITLE
Component: Dialogue

### ISSUE THREAD

**Issue by jnschrag** [CSIS-iLab/ocean#63]
Displays dialogue between participants. This should be able to account for multiple participants. My recommendation is that this component is implemented in regular HTML whenever it's used, following this model:

```html
<div class="dialogue">
<blockquote>
<cite>Speaker One</cite>
Speaker one says something here
</blockquote>

<blockquote>
<cite>Speaker One</cite>
Speaker one says something here
</blockquote>
</div>
```

The only "component" implementation should be the CSS styles.

**Comment by jnschrag** [comment]
@aleesteele Re the description post, you should use those HTML elements and modify their styles specifically for their implementation in the Spotlights. You should not edit the base styles in `_typography.scss`, but rather add on to/override them for the implementation as a part of the dialogue component like you've started to do in the draft version of the component.

**Comment by jnschrag** [comment]
Closed by #224
